package.json

```diff
@@ -27,9 +27,9 @@
     "react-dom": "^0.14.7"
   },
   "devDependencies": {
+    "babel-core": "^6.4.5",
     "babel-loader": "^6.2.2",
     "babel-preset-es2015": "^6.3.13",
-    "babel-preset-es2015-loose": "^7.0.0",
```

Comment:
I'm not sure about that. But either way I think for the beginning of a project Spec Correctness is more important then small perf hits that you aren't sure about. Perf can always be tweaked, I say use the 'official' stuff first and mess with config later, once you have something working and need to start squeezing perf